public/script.js

```diff
@@ -458,7 +458,9 @@ let currentVersion = '0.0.0';
 export const default_ch_mes = 'Hello';
 let generatedPromptCache = '';
 let generation_started = new Date();
+/** @type {v1CharData[]} */
```

Comment:
To use JSDoc types, you need to import them: `/** @type {import('scripts/char-data.js').v1CharData[]} */`

Reply:
fixed in https://github.com/SillyTavern/SillyTavern/commit/3988038623854b1871035391563e2664b32d27db